src/client/client.rs

```diff
@@ -571,43 +576,51 @@ mod tests {
 
     #[test]
     fn personal_urls() {
-        let urls = Urls::from_account(&Account::User);
+        let urls = Urls::new(
+            &Url::try_from(Input::DEFAULT_GITHUB_SERVER_URL).unwrap(),
+            &Url::try_from(Input::DEFAULT_GITHUB_API_URL).unwrap(),
+            &Account::User,
+        );
         assert_eq!(
             urls.list_packages_url.as_str(),
-            "https://api.github.com/user/packages?package_type=container&per_page=100"
+            Input::DEFAULT_GITHUB_API_URL.to_string() + "/user/packages?package_type=container&per_page=100"
         );
         assert_eq!(
             urls.list_package_versions_url("foo").unwrap().as_str(),
-            "https://api.github.com/user/packages/container/foo/versions?per_page=100"
+            Input::DEFAULT_GITHUB_API_URL.to_string() + "/user/packages/container/foo/versions?per_page=100"
         );
         assert_eq!(
             urls.delete_package_version_url("foo", &123).unwrap().as_str(),
-            "https://api.github.com/user/packages/container/foo/versions/123"
+            Input::DEFAULT_GITHUB_API_URL.to_string() + "/user/packages/container/foo/versions/123"
         );
         assert_eq!(
             urls.package_version_url("foo", &123).unwrap().as_str(),
-            "https://github.com/user/packages/container/foo/123"
+            Input::DEFAULT_GITHUB_SERVER_URL.to_string() + "/user/packages/container/foo/123"
         );
     }
 
     #[test]
     fn organization_urls() {
-        let urls = Urls::from_account(&Account::Organization("acme".to_string()));
+        let urls = Urls::new(
+            &Url::try_from(Input::DEFAULT_GITHUB_SERVER_URL).unwrap(),
+            &Url::try_from(Input::DEFAULT_GITHUB_API_URL).unwrap(),
+            &Account::Organization("acme".to_string()),
+        );
         assert_eq!(
             urls.list_packages_url.as_str(),
-            "https://api.github.com/orgs/acme/packages?package_type=container&per_page=100"
+            Input::DEFAULT_GITHUB_API_URL.to_string() + "/orgs/acme/packages?package_type=container&per_page=100"
         );
         assert_eq!(
             urls.list_package_versions_url("foo").unwrap().as_str(),
-            "https://api.github.com/orgs/acme/packages/container/foo/versions?per_page=100"
+            Input::DEFAULT_GITHUB_API_URL.to_string() + "/orgs/acme/packages/container/foo/versions?per_page=100"
```

Comment:
Wouldn't this be  
```
https://api.github.com//orgs/acme/packages/container/foo/versions?per_page=100
                      ^ two `//`'s here
```
considering you add a `/` in the `Url::parse((url_str.to_string() + "/").as_str())`?

Reply:
Oh, I now see that you removed the prefix from the URLs in the code, and just read your previous comment about slashes. So this is probably not an issue, then?

Reply:
Yes, it's not an issue, the tests would have failed otherwise. Adding a slash is only for the conversion to `Url`, since the `url` library adds them (but only on an empty path).